pyproject.toml

```diff
@@ -117,6 +117,7 @@ nc4-cm1 = ["yt[netCDF4]"]
 open-pmd = ["yt[HDF5]"]
 owls = ["yt[HDF5]"]
 owls-subfind = ["yt[HDF5]"]
+parthenon = ["yt[HDF5]"]
```

Comment:
I can't find where h5py is actually used in the frontend, could you clarify the intent here ?

Reply:
We directly use  `self._handle = HDF5FileHandler(filename)` rather than explicit calls to `h5py`.
Does this implicit use mean, `hdf5` should not be set as a dependency?
Parthenon currently only supports output in HDF5 so we definitely rely on this lib/module.

Reply:
`HDF5FileHandler` indeed hard-requires `h5py` so you are absolutely right to list it as a dependency, I just missed that. Thank you !